README.md

```diff
@@ -8,7 +8,14 @@ Android Studio users see [transition guide](ANDROID_README.md).
 
 # Requirements
 
-[Gradle](http://gradle.org) is required to build and run the plugin.
+[Gradle](http://gradle.org) is required to build and run the plugin. The table below shows the compatibility with Gradle version.
+
+<!-- TODO: Remove "(not yet released)" once we release a new version -->
+
+| Gradle version | endpoints-framework-gradle-plugin version |
+| ------------- | ------------- |
+| 7.x - | 2.2.0 or higher (not yet released) |
+| 4.x - 6.x | 2.1.0 or lower |
```

Comment:
@loosebazooka Does this look good?

Reply:
I mean there should be a way to build with gradle 6.x and be compatible with 7.x? Just need to make sure the new code is compatible with both 6.x and 7.x api?

Reply:
The new code works with Gradle 5.1 onwards, including 7.x.

https://github.com/GoogleCloudPlatform/endpoints-framework-gradle-plugin/pull/109#commitcomment-53170886

Reply:
I'd say there are two directions. With 2.2.0, you can continue to use old Gradle versions. With Gradle 7+, however, you must use 2.2.0 (old plugin versions won't work).

Reply:
> With Gradle 7+, however, you must use 2.2.0 (old plugin versions won't work)

Yes, this is the message of the table.